### PR TITLE
[docs] - Add new Schedules concept page (DOC-193)

### PR DESCRIPTION
**Pull request by erinkcochran87**
## Summary & Motivation

This PR adds a new concept page for Schedules. The goal is for this page to explain what a schedule is and why it's beneficial, not to walk users through how to create/use it.

## How I Tested These Changes

eyes, local

### CODE REVIEW

**Comment by erinkcochran87** [comment]
@tacastillo @cmpadden This is admittedly pretty rough, but I've been staring at it for too long and could use some feedback 🙏

File: docs/content/concepts/automation/schedules.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: "Schedules | Dagster Docs"
+description: "Use schedules to run your pipelines at fixed time intervals."
+---
+
+# Schedules
+
+Schedules are Dagster's way of supporting traditional methods of [automation](/concepts/automation), which allow you to specify when a [job](/concepts/ops-jobs-graphs/jobs) should run. Using schedules, you can define a fixed time interval to run a pipeline, such as daily, hourly, or Monday at 9:00 AM.
+
+Each interval of a schedule is called a **tick**, which is an indication that a job should begin executing. Ticks kick off **runs**, which is a single instance of a job being executed.
```
Comment (cmpadden) [inline docs/content/concepts/automation/schedules.mdx:10]: If this is the same "tick" as [freshness policy sensors](https://docs.dagster.io/concepts/partitions-schedules-sensors/asset-sensors#freshness-policy-sensors), I feel like my understanding is our "scheduler" has a tick, every <n> seconds, and in each tick it checks to see if the schedule should run, then if that tick occurs at the time indicated by the schedule, a run will be launched.

Does that align with this sentence?

File: docs/content/concepts/automation/schedules.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- Consistently run data pipelines without the need for manual intervention
+- Optimize resource usage by scheduling pipelines to run during off-peak hours
+
+---
+
+## Prerequisites
+
+Before continuing, you should be familiar with:
+
+- [Cron syntax](https://en.wikipedia.org/wiki/Cron)
```
Comment (cmpadden) [inline docs/content/concepts/automation/schedules.mdx:30]: Might be worth sharing here, but when dealing with Cron I almost always use https://crontab.guru/

**Review by cmpadden — APPROVED** [review]
Nice, this looks good! Left one comment regarding "ticks".

File: docs/content/concepts/automation/schedules.mdx (hunk 1 above)
Comment (erinkcochran87) [reply docs/content/concepts/automation/schedules.mdx:10]: I'm inclined to say no - it's my understanding that whenever the current time aligns with a schedule's interval, a tick is created and then a run is launched.

The **Tick history** tab in the UI seems to support this: 
Screenshot 2024-05-31 at 10 45 10 AM

Comment (cmpadden) [reply docs/content/concepts/automation/schedules.mdx:10]: Gotcha thank you for the clarification -- that makes sense to me.

**Pull request merged**